### PR TITLE
Validation pipeline pr curve

### PR DESCRIPTION
**Pull request by erickzul**
Graph: https://screenshot.googleplex.com/BmhNRAFAvKEUV9T

### CODE REVIEW

File: validation_pipeline.ipynb [hunk 1]
```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
```
Comment (rothn) [inline validation_pipeline.ipynb:7]: <p>Are 'fraud' and 'predicted_fraud' class probabilities somehow? I am having trouble understanding how that could be the case.</p><p> </p><p><a href="https://scikit-learn.org/stable/modules/generated/sklearn.metrics.PrecisionRecallDisplay.html#sklearn.metrics.PrecisionRecallDisplay.from_predictions" target="_blank">https://scikit-learn.org/stable/modules/generated/sklearn.metrics.PrecisionRecallDisplay.html#sklearn.metrics.PrecisionRecallDisplay.from_predictions</a> <code>y_true array-like of shape (n_samples,) True binary labels. y_pred array-like of shape (n_samples,) Estimated probabilities or output of decision function.   </code> </p><p> </p><p>I would recommend reusing the results of the p-value sweep you did for the accuracy plot and then plotting precision against recall with pyplot&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

**Review by rothn — CHANGES_REQUESTED** [review]
See comment

File: validation_pipeline.ipynb (hunk 1 above)
Comment (rothn) [reply validation_pipeline.ipynb:7]: So if "fraud" is 1/0 labels and "predicted_fraud" are p-values, I can see how this would make sense. The p-value threshold would take the place of where I usually would see the class probability. If so, can you confirm this is the case, and if so, write a short explanation to this point?

Comment (rothn) [reply validation_pipeline.ipynb:7]: If not, please still let me know! And would you also mind putting a P-R curve in the PR?

Comment (erickzul) [reply validation_pipeline.ipynb:7]: It should be there I really don't know why it doesn't show in the diffs:
https://screenshot.googleplex.com/BmhNRAFAvKEUV9T

Per https://screenshot.googleplex.com/8gphW7cydwLeBEB, predicted_p_value when low should be positive (fraud).

Comment (erickzul) [reply validation_pipeline.ipynb:7]: This was already how it was implemented before making this change.

Comment (erickzul) [reply validation_pipeline.ipynb:7]: https://screenshot.googleplex.com/B94r4ByyBC5qCET

This is how it was implemented on main before this change. I don't think we invert any values.

**Review by rothn — APPROVED** [review]
Approved after f2f
